### ISSUE TITLE
Add support for "#EXTALBUMARTURL:" tag

### ISSUE THREAD

**Issue by lars18th** [xteve-project/xTeVe#350]
**Is your feature request related to a problem? Please describe.**
Some programs (like VLC) read the channel logo using the tag `#EXTALBUMARTURL:`. So I suggest to add this tag in the output M3U playlist.

**Describe the solution you'd like**
It's very simple. If you have the logo URL, after the line `#EXTINF:   tvg-logo="<logo_url>"` you add in the next line
`#EXTALBUMARTURL:<logo_url>` 

**Describe alternatives you've considered**
No alternatives except editing the M3U manually.

**Additional context**
The VLC is a good client for testing the M3U. Or to show it using a PC. So it has sense to support it.

**Comment by MCTyler** [comment]
We don't intend to support VLC as a Media Player because it does not support EPG. Check with one of the Forks.

**Comment by lars18th** [comment]
Hi @MCTyler ,

> We don't intend to support VLC as a Media Player because it does not support EPG

The idea is not to target VLC as a _regular_ media player. It's only to add support for more compatible tags. Therefore as the `#EXTALBUMARTURL` tag is not intrusive (it doesn't change anything as any M3U reader discards unsupported tags) and it's very easy to add it (because the <logo_url>) then this support is very straightforward.

In fact to support it only one new line is required between: https://github.com/xteve-project/xTeVe/blob/0e999b85b9c77c3fdcfcfaf5e96150523fd4481b/src/m3u.go#L228-L232
So you accept to include it if I'll prepare a PR with this change?

**Comment by MCTyler** [comment]
No, The Dev has retired and we are making no changes to our code. That's why I said find a fork and see if they will include your request.

**Comment by lars18th** [comment]
Hi @MCTyler ,

Thank you for the response. So this project is closed. Which fork then you recomend?

**Comment by MCTyler** [comment]
I don't recommend any fork. The project is "complete" not closed.

**Comment by lars18th** [comment]
> I don't recommend any fork. The project is "complete" not closed.

Umm... I don't agree with that! Any project not accepting PRs is dead. For example, look out for pending patches that fix some issues or add new features in this repository.

Perhaps, it will be useful if some users will generate a new fork and apply all pending patches to it.
What you think?
